demos/human_pose_estimation_demo/cpp/main.cpp

```diff
@@ -343,8 +342,18 @@ int main(int argc, char *argv[]) {
         }
 
         //// --------------------------- Report metrics -------------------------------------------------------
-        slog::info << slog::endl << "Metric reports:" << slog::endl;
+        slog::info << "Metrics report:" << slog::endl;
         metrics.printTotal();
+        slog::info << "\tDecoding:\t" << std::fixed << std::setprecision(2) <<
```

Comment:
The logging of preprocess/infer/postprocess metrics might be moved in a separate function, for example 
`void reportMetrics(ImagesCapture cap, AsyncPipeline pipeline, PerformanceMetrics renderMetrics)`